src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
```

Comment:
remove this array, you don't need it, just call getRandomBall method inside of the for loop

Reply:
[balls array removed](https://github.com/Maksym-Kulakov/jv-lottery/commit/9264cec11d75f0d049fe07b253628ab0db56fc4a)